app/views/users/_listuser.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="profIntro">
+  <img src="<%= user.photo %>" alt="profileImg" class="profileImg">
+  <div>
+    <h3><%= user.name %></h3>
+    <span> Number of Posts:<%= user.posts_counter %></span>
+  </div>
```

Comment:
Nice job so far👍🏼. You are meant to make this a link or add a link that leads the user to the user show page (each user rendered should link to their respective show page)😉. please do well to ensure your views match the wireframe as this is a requirement👍🏼.

yours
<img width="1354" alt="image" src="https://github.com/GeekyHacks/FlightOperationsBlog/assets/64643995/0c128337-c4ce-4772-8191-934804ca4a0d">

wireframe
<img width="1013" alt="image" src="https://github.com/GeekyHacks/FlightOperationsBlog/assets/64643995/99b93672-b10a-4413-8746-8b2715589d1a">


<img width="991" alt="image" src="https://github.com/GeekyHacks/FlightOperationsBlog/assets/64643995/ef45e26e-ca83-4682-9c05-cedac52c9238">



As regards the wireframe, this is the same for all your pages, try to ensure your views match the wireframes and the necessary links to the corresponding views are provided😉😉